### PR TITLE
Addition of `systemctl` command in Linux directory in DevOps repo

### PR DESCRIPTION
**Pull request by ShubhamJagtap2000**
## 🛠️ Fixes Issue

#82 

## 👨‍💻 Changes proposed

Added a bulleted chunk for the `systemctl` command and also options that we could use to interact with process in Linux 
## ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## 📄 Note to reviewers
Please check the below screenshot to see the preview of the change.

## 📷 Screenshots
image

<a href="https://gitpod.io/#https://github.com/Pradumnasaraf/DevOps/pull/83"></a>

### CODE REVIEW

File: Linux/commands/README.md [hunk 1]
```diff
@@ -57,6 +57,13 @@
 
 ### System commands
 - `ps aux` - processes which are running
+- `systemctl [option] [service]` - interact with a process
+    - We can do 4 `option` with `systemctl`
+        - start
+        - stop
+        - enable
+        - disable
```
Comment (ShubhamJagtap2000) [inline Linux/commands/README.md:65]: Earlier by mistake, I had added it as 'disabel' but after raising the PR, I have again fixed it to 'disable'. Thank you.

**Comment by ShubhamJagtap2000** [comment]
Hello @Pradumnasaraf, were you able to review the PR?

**Review by Pradumnasaraf — COMMENTED** [review]
Hey,

**Comment by ShubhamJagtap2000** [comment]
Really? Let me check again...

**Comment by ShubhamJagtap2000** [comment]
> Hey,
> 
> You are violating the https://github.com/Pradumnasaraf/DevOps/blob/main/CONTRIBUTING.md. Last time I merged your PR.

Hi, can you help me with what is getting violated? Is that issue assignment?

**Review by Pradumnasaraf — COMMENTED** [review]
> Is that about issue assignment?

Yes

**Review by Pradumnasaraf — APPROVED** [review]

**Comment by ShubhamJagtap2000** [comment]
> > Is that about issue assignment?
> 
> Yes

Thanks, will take approval from you before raising the PR going forward.

**Pull request merged**